packages/flutter_tools/test/data/dart_dependencies_test/syntax_error/foo.dart

```diff
@@ -2,4 +2,4 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-bad programmer!
+import bad programmer!
```

Comment:
Ah that makes sense. I was confused as to which test this is (we have a bevy of tests that verify handling of bad syntax, it turns out.)